portfolio/src/main/webapp/scripts/login.js

```diff
@@ -16,12 +23,17 @@ async function showCommentFeedback() {
  * Changes button text to reflect whether user should log in or out
  */
 async function updateLoginStatusOnButton() {
+  const loginButton = document.getElementById('login-button'); 
   try {
     const response = await fetch('/check-login-status');
```

Comment:
You don't need to make this call here.  The only time this method gets called is from showCommentFeedback where you already have this information.

In fact these could probably be combined into one method.